com.tracktion.Waveform11.yml

```diff
@@ -0,0 +1,128 @@
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --filesystem=xdg-config
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=xdg-videos
+  - --filesystem=/run/media
+  - --filesystem=/media
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+
+cleanup:
+  - '*.a'
+  - '*.la'
+  - '/lib/libjack*'
+  - /lib/jack
+  - /include
+  - /cache
+  - /include
+
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '20.08'
+    add-ld-path: lib
+    merge-dirs: ladspa;lv2;lxvst;vst3
+    subdirectories: true
+    no-autodownload: true
+
+modules:
+  - shared-modules/linux-audio/jack2.json
+
+  - name: x264
+    config-opts:
+      - --prefix=/app
+      - --disable-cli
+      - --enable-shared
+    cleanup:
+      - /include
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        commit: 55d517bc4569272a2c9a367a4106c234aba2ffbc
+
+  - name: ffmpeg
+    config-opts:
+      - --enable-gpl
+      - --enable-shared
+      - --disable-static
+      - --enable-gnutls
+      - --disable-doc
+      - --disable-programs
+      - --disable-devices
+      - --enable-libopus
+      - --enable-libvpx
+      - --enable-libvorbis
+      - --enable-libx264
+    cleanup:
+      - /share/ffmpeg
+      - /include
+    sources:
+      - type: archive
+        url: https://www.ffmpeg.org/releases/ffmpeg-4.4.tar.xz
+        sha256: 06b10a183ce5371f915c6bb15b7b1fffbe046e8275099c96affc29e17645d909
+
+  - name: xjadeo
+    cleanup:
+      - /share/man
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/xjadeo/xjadeo/v0.8.9/xjadeo-0.8.9.tar.gz
+        sha256: 47b7aeb9d0a82178a72944bac31b3934afee620fb05eefb625f6f3f085d266f1
+
+      - type: patch
+        path: patches/xjadeo-c-fix.patch
+
+  - name: xprop
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xprop-1.2.4.tar.bz2
+        sha256: 8c77fb096e46c60032b7e2bde9492c3ffcc18734f50b395085a5f10bfd3cf753
+
+  - name: waveform11
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 /usr/bin/desktop-file-edit -t /app/bin/
+      - install -Dm755 /usr/bin/ar -t /app/bin/
+      - install -Dm755 /usr/lib/$(gcc -print-multiarch)/libbfd-*.so -t /app/lib/
+      - install -Dm644 com.tracktion.Waveform11.appdata.xml ${FLATPAK_DEST}/share/appdata
+      - install -Dm755 apply_extra.sh ${FLATPAK_DEST}/bin/apply_extra
+      - install -Dm755 waveform11.sh ${FLATPAK_DEST}/bin/waveform11
+    post-install:
+      - install -d /app/extensions/Plugins
+    sources:
+      - type: extra-data
+        filename: waveform11.deb
+        only-arches:
+          - x86_64
+        url: https://cdn.tracktion.com/file/tracktiondownload/waveform/1159/waveform_64bit_v11.5.9.deb
```

Comment:
you need to add the exclusion in the flathub.json.

Reply:
Good point